src/components/calcite-chip/calcite-chip.scss

```diff
@@ -1,109 +1,15 @@
-// scale
-// todo update when new spacing modifiers are introduced to calcite-base
-:host([scale="xs"]) {
-  font-size: 10px;
-  --calcite-chip-spacing-unit-l: 8px;
-  --calcite-chip-spacing-unit-s: 4px;
-}
-:host([scale="s"]) {
-  font-size: 12px;
-  --calcite-chip-spacing-unit-l: 12px;
-  --calcite-chip-spacing-unit-s: 8px;
-}
-
-:host([scale="m"]) {
-  font-size: 14px;
-  --calcite-chip-spacing-unit-l: 16px;
-  --calcite-chip-spacing-unit-s: 12px;
-}
-
-:host([scale="l"]) {
-  font-size: 16px;
-  --calcite-chip-spacing-unit-l: 20px;
-  --calcite-chip-spacing-unit-s: 16px;
-}
-
-:host([scale="xl"]) {
-  font-size: 18px;
-  --calcite-chip-spacing-unit-l: 24px;
-  --calcite-chip-spacing-unit-s: 20px;
-}
-
 :host {
-  --calcite-chip-button-border-radius: 0 50px 50px 0;
-  display: inline-flex;
-  align-items: center;
-  justify-self: center;
-  background-color: var(--calcite-ui-background);
-  border-radius: 50px;
-  color: var(--calcite-ui-text-1);
-  font-weight: 500;
+  @apply inline-flex items-center rounded-full font-medium text-black bg-gray-lightest;
```

Comment:
The only styles that ended up remaining in the `scss` file were those needed to style the host element or slotted elements.